### PR TITLE
Connections: Change Instagram "Widget" to "Block"

### LINKED ISSUE #66730: Connections: Change Instagram "Widget" to "Block"
### Quick summary

The Instagram _widget_ no longer exists, but there is a reference to it in Calypso.

### Steps to reproduce

1. Go to  Tools → Marketing → Connections 
2. Expand Instagram

It says:

>Add an Instagram widget to display your latest photos.

image

### What you expected to happen

It should say "block".

### What actually happened

It says "widget", which is outdated.

### Context

_No response_

### Platform (Simple, Atomic, or both?)

Simple, Atomic

### Theme-specific issue?

_No response_

### Browser, operating system and other notes

_No response_

### Reproducibility

Consistent

### Severity

All

### Available workarounds?

No but the platform is still usable

### Workaround details

_No response_

### PR DESCRIPTION
**Pull request by katinthehatsite**
Closes https://github.com/Automattic/wp-calypso/issues/66730

## Proposed Changes

This PR changes the mention of `Instagram Widget` to `Latest Instagram Posts` block.

## Testing Instructions

* Navigate to the Calypso Live Link: http://calypso.localhost:3000/marketing/connections/{yourSite}
* Scroll to the bottom of the page to the Instagram section
* Confirm that the text says: `Add the Latest Instagram Posts` block and the link goes to: https://wordpress.com/support/instagram/#embed-a-feed-of-instagram-posts

Screenshot 2024-09-06 at 2 01 44 PM

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/marketing/connections/service-description.jsx [hunk 1]
```diff
@@ -154,14 +154,8 @@ class SharingServiceDescription extends Component {
 				);
 			},
 			instagram_basic_display: function () {
-				if ( this.props.numberOfConnections > 0 ) {
-					return this.props.translate( 'Connected to your Instagram account.', {
-						comment: 'Description for Instagram when one or more accounts are connected',
-					} );
-				}
-
-				return this.props.translate( 'Connect to use the Instagram widget.', {
```
Comment (katinthehatsite) [inline client/my-sites/marketing/connections/service-description.jsx:169]: I also updated connection description and removed the widget wording from there.

**Comment by obenland** [comment]
When the strings are final, could you add the string freeze label so they can be translated before merging the PR? It helps avoid untranslated strings in an otherwise translated interface.

File: client/my-sites/marketing/connections/service-description.jsx (hunk 1 above)
Comment (wojtekn) [reply client/my-sites/marketing/connections/service-description.jsx:169]: How does the description behave after removing the condition?

**Comment by wojtekn** [comment]
@katinthehatsite, the widget ->block change works for me, and the code change looks clear.

I left a comment related to the other change.

File: client/my-sites/marketing/connections/service-description.jsx (hunk 1 above)
Comment (katinthehatsite) [reply client/my-sites/marketing/connections/service-description.jsx:169]: @wojtekn before this change, this is how it worked:

* if the user had no accounts connected yet, the would see the text `Connect to use the Instagram widget`
* if they had one other account connected, they would see the text `Connected to your Instagram account.`

Please note that you can connect more than one Instagram account. 

After the change, the users would see `Connected to your Instagram account` for both cases. I think this change would be okay as Instagram widget does not really exist anymore.

Comment (wojtekn) [reply client/my-sites/marketing/connections/service-description.jsx:169]: Those subtitles inform users what they can do with a particular connection:

Screenshot 2024-09-13 at 12 49 58

'Connected to your Instagram account' doesn't seem right now. What if we changed it to "Display latest posts from your Instagram profile."?

Should we also remove the 'Connect' button?

Comment (katinthehatsite) [reply client/my-sites/marketing/connections/service-description.jsx:169]: Let me test a bit more: I am hesitant to make bigger changes without design input but I am going to check a couple of things first 👀

Comment (wojtekn) [reply client/my-sites/marketing/connections/service-description.jsx:169]: @katinthehatsite, the text is wrong now, and "Connect" opens the broken page, so I think we can proceed and fix those. Or am I missing something?

Comment (katinthehatsite) [reply client/my-sites/marketing/connections/service-description.jsx:169]: The text is wrong for me as well but the button works as expected.  I can connect and disconnect on my end:

Screenshot 2024-09-23 at 7 50 46 PM

I will make some changes first and then ask for review and we can go from there.

Comment (katinthehatsite) [reply client/my-sites/marketing/connections/service-description.jsx:169]: @wojtekn I looked a bit more into this and I propose the following:

* https://github.com/Automattic/wp-calypso/pull/94284/commits/12dbf3591e272961eb5b9746aff50ff6bf844018 - In this commit, I reverted the change where I removed the condition to check how many accounts are connect so the text for that should be back in place and displaying correctly
* https://github.com/Automattic/wp-calypso/pull/94284/commits/c55d845d7e7f7f953e61d0b1c7f46d5e8eb1e5aa - In this commit, I changed the wording from `Instagram widget` to `Latest Instagram Posts block`. From my testing, it seems that the Instagram connection is required to use specifically that block
* I am not sure why we would need to remove the `Connect` button as it works as expected for me for both an Atomic and a Simple site

What do you think?

Comment (wojtekn) [reply client/my-sites/marketing/connections/service-description.jsx:169]: Thanks @katinthehatsite , I think it's good now.

I missed that the new block still requires the connection, and at the time of my test, 'Connect' was showing a page with the error, so it somehow convinced me that this is how it's supposed to work.

**Review by wojtekn — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16880414

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @katinthehatsite for including a screenshot in the description! This is really helpful for our translators.

**Comment by katinthehatsite** [comment]
Thanks @wojtekn - I added the strings freeze label and once the strings are translated, I will merge this PR 👍

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16880414).

**Comment by wojtekn** [comment]
@katinthehatsite any idea what happened here? Bot said that translations are finished, but the CI check is marked as failed and shows translations are 91% done. 🤔

**Comment by katinthehatsite** [comment]
@wojtekn yeah, not sure. I saw the same issue yesterday so I was waiting to see if it gets resolved with something perhaps being out of sync. I will check 🤔

**Comment by katinthehatsite** [comment]
After checking with the team responsible for translations, I believe we can proceed with deploying as all strings are present (although some of them are fuzzy) 🚢

**Pull request merged**